### DISCUSSION TITLE
Disable CSRF

### DISCUSSION THREAD

**Discussion by larsjanssen6** [statamic/cms#5934] [General]
Hi, 

Is it possible to completely disable CSRF in statamic. I already disabled it in `kernel.php` but that does not work. 

I know it's a security issue if you disable this, but for a local system I need to do this. 

Can this be done?

Lars

**Comment by duncanmcclean** [comment]
You can add a wildcard (`/*`) to the array in your `app/Http/Middleware/VerifyCsrfToken.phg` file.

However, like you say, it's highly recommended that you do use CSRF.

**Reply by larsjanssen6** [reply]
Thanks!

**Comment by edalzell** [comment]
Don't do this. Why do you need to do it locally?
